src/Discord.Net/Entities/Activities/IActivity.cs

```diff
@@ -0,0 +1,10 @@
+// todo: meth
```

Comment:
Don't think that's good for your health.

Reply:
Unless you have lung cancer.